src/Parser/DeathKnight/Unholy/Modules/Features/DarkTransformationAndWounds.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import SpellLink from 'common/SpellLink';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+import { encodeTargetString } from 'Parser/Core/Modules/EnemyInstances';
+import Wrapper from 'common/Wrapper';
+
+class DarkTransformationAndWounds extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+  // used to track how many stacks a target has
+  targets = {};
+
+  darkTransformationActive = 0;
+  darkTransformationCasts = 0;
+  stopWatchActive = 0;
```

Comment:
you should probably use `true` and `false` instead of `1` and `0` to make it more obvious that `darkTransformationActive` and `stopWatchActive` they are bools.